.github/CONTRIBUTING.md

```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
+- Revise pull requests according to review feedback. We will close pull requests that require an inordinate amount of time to review and process (especially for minor changes) if you fail to make revisions according to review feedback.
+- Do not directly contact DevDocs team members or maintainers on Slack to review your pull request unless it has been open for more than five days. We have a process and queue for pull requests that everyone must follow.
 
-Focus on the content with useful information, code samples, and important notes for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+## Tips for writing content
 
 Use the following guidelines to help you with the writing process:
 
+- Focus your efforts on providing useful information for your fellow Magento developers and community members. For example, new or revised code samples, important notes, clarification of vague or ambiguous content.
 - Define the goal of your topic. What exactly do you want to teach the reader?
 - Make the title of your topic reflect the content.
 - Keep your sentences concise. Separate conceptual information from procedural steps.
-- Batch several small changes into a single pull request (instead of separate ones) to ensure your contributions get approved and merged quickly. Have several typo fixes across several areas of documentation? Batch them into one PR.
-- Remember to use active voice (not passive), write in the present tense, and use a friendly tone in second person. For example, _"The log captures commands, output..."_.
+- Batch several small changes into a single pull request (instead of separate ones) to ensure your contributions are approved and merged quickly. Have several typo fixes across several areas of documentation? Combine them into a single PR.
+- Remember to write in present tense, use the second person, and use active voice (not passive). For example, _"The log captures commands, output..."_.
 - Use notes to alert readers about important details.
-- Use cross-references to other topics if appropriate. We can help you with the syntax if it is not clear. The template provides an example you can use.
+- Use cross-references to other topics sparingly. We can help you with the syntax if it is not clear. The template provides an example you can use.
 - Need help with markdown? See our [templates](#templates).
 
-### Preview your work on local
+### Review changes locally
 
-Optional. To preview your changes in HTML output, follow the instructions in the [README](https://github.com/magento/devdocs/blob/master/README.md) to build the devdocs site locally using Jekyll.
+_(Optional)_ To review your changes in HTML output, follow the instructions in the [README](https://github.com/magento/devdocs/blob/master/README.md) to build the devdocs site locally using Jekyll.
```

Comment:
Not sure if will work. The README does not contain any instructions about how to build and preview subrepositories.

Reply:
Contributors may not be able to build some repos even after we update the README since some repos are private.